src/UrlParams.ts

```diff
@@ -124,9 +124,15 @@ export interface UrlParams {
    */
   password: string | null;
   /**
-   * Whether we the app should use per participant keys for E2EE.
+   * Whether the app should use per participant keys for E2EE.
    */
   perParticipantE2EE: boolean;
+  /**
+   * Whether the global JS controls for audio output devices should be enabled,
+   * allowing the list of output devices to be controlled by the app hosting
+   * Element Call.
+   */
+  controlledMediaDevices: boolean;
```

Comment:
By the way, maybe this should instead be named `controlledOutputDevices` to reflect its behavior? I think I'm responsible for that mis-naming.

Reply:
Different renaming suggested at https://github.com/element-hq/element-call/pull/3270#discussion_r2101824073